packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -32,4 +32,28 @@ UPLOADCOLLECTION_NO_DATA_DESCRIPTION=Drop files to upload, or use the Upload but
 UPLOADCOLLECTION_DRAG_FILE_INDICATOR=Drag files here
 
 #XMSG: Message text indicating where to drop file and upload
-UPLOADCOLLECTION_DROP_FILE_INDICATOR=Drop files to upload
\ No newline at end of file
+UPLOADCOLLECTION_DROP_FILE_INDICATOR=Drop files to upload
+
+#XACT: ARIA announcement for the logo button
+SHELLBAR_LABEL = Logo
```

Comment:
This is the right place for these however.

Reply:
Done